modules/ROOT/pages/policies-landing-page.adoc

```diff
@@ -4,7 +4,46 @@ include::_attributes.adoc[]
++
+You can apply a policy to any API implemented with Mulesoft for any of the proxy types: RAML/Rest, HTTP, or SOAP-based as long as the proxy is exposed through an HTTP endpoint.
+You can also apply a policy to APIs not implemented with Mulesoft by deploying a Proxy application to control how and when a received request is forwarded to its implementation endpoint.
+
+You can set an API alert to notify you when an API request violates a policy for SLA. By default, a policy applies to the entire API, filtering traffic requests to every resource and method. In Mule Runtime 3.8.0 and later, you can enhance security through policies by using Gatekeeper. Gatekeeper disables an API until all online policies are applied.
+
+== Policy Architechture
+
+Policies are implemented through coordinated communication between the following components:
+
+* API Manager
+* One or more API Gateway Runtimes or Mule 3.8.0 Runtime or later
+* One or more API proxy applications
+
+Using API Manager, you can configure and apply policies to an API instance. An API version can have one or more API instances. +
+A specific API instance can also be tied to a specific API implementation endpoint, which can then auto-generate an API proxy application. +
+You can deploy an API Proxy to a Mule runtime.
+
+Each API proxy application receives requests on HTTP or HTTPS URLs specified by the API. Normally requests are forwarded to the corresponding API implementation endpoint URL, and then the response travels through the API proxy application back to the requesting client application.
+
+== Policy Injection and Enforcement
+
+When you deploy an API proxy application for a specific API version to a Mule Runtime, any applied policies in the runtime or policies folder are injected into the API proxy. This changes the behavior of the application. When the proxy application receives a new request, all injected policies are applied to decide if, and how, the request is forwarded to the API implementation endpoint.
+
+In this way, the actual policy enforcement occurs inside the proxy application itself, minimizing the cross talk between the API proxy, which is processing the received request, the Anypoint Platform agent, and online API Manager. The API proxy application does not require any communication with the Anypoint Platform agent running in the runtime, nor does it require any communication with API Manager.
+
+However, the Anypoint Platform agent remains connected to API Manager. After policies are reconfigured or removed from API Manager, those policies are downloaded to any connected API Gateway or Mule runtimes, which updates each runtime /policies folder. The policy changes are then again injected into each API proxy application. This allows policies to be dynamically changed without having to redeploy the API proxy application, and without having to restart the API Gateway or Mule runtimes.
+
 
 
 == See Also
```

Comment:
We should refer to gatekeeper documentation if it's not there already: https://docs.mulesoft.com/api-manager/2.x/gatekeeper

Reply:
Done! Thanks!